test/e2e/tests/confirmations/alerts/insufficient-funds.spec.ts

```diff
@@ -15,21 +10,16 @@ import { Driver } from '../../../webdriver/driver';
 describe('Alert for insufficient funds', function () {
   it('Shows an alert when the user tries to send a transaction with insufficient funds', async function () {
     const nftSmartContract = SMART_CONTRACTS.NFTS;
-    const ganacheOptions = {
-      accounts: [
-        {
-          secretKey: PRIVATE_KEY,
-          balance: convertETHToHexGwei(0.0053), // Low balance only to create the contract and then trigger the alert for insufficient funds
-        },
-      ],
+    const localNodeOptions = {
+      mnemonic: 'test test test test test test test test test test test junk',
```

Comment:
anvil doesn't handle setting balance with priv key, we just use a different mnemonic, so the account swith fixtures will have already 0 balance